src/Factory/AuthControllerFactory.php

```diff
@@ -31,24 +30,6 @@ public function __invoke(ContainerInterface $container, $requestedName, ?array $
         return $authController;
     }
 
-    /**
-     * @param null|string $name
-     * @param null|string $requestedName
-     * @return AuthController
-     */
-    public function createService(ServiceLocatorInterface $controllers, $name = null, $requestedName = null)
```

Comment:
@snapshotpl if we keep the BC breaks in, we need a major: can we keep the dead `public` methods for a while, and mark them `@deprecated` instead?